### PR TITLE
pin colander < 2

### PR DESCRIPTION
**Pull request by daeMOn63**
With the recent release of Colander 2.0, we need the current colander-tools version to pin colander < 2 because of a `colander.Mapping._impl()` signature change (it now takes 5 params instead of 4), as well as a dropped support for python <= 3.6.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1 +1 @@
--e .
```
Comment (danmilon) [inline requirements.txt]: weird change, is it whitespace / new line? lets take it out anyway. and squash commits?

Comment (daeMOn63) [reply requirements.txt]: ah right, the newline got lost there, thanks, restored now

**Comment by danmilon** [comment]
@damz lets merge this and release 0.12.1, and then merge https://github.com/platformsh/colander-tools/pull/17 and release 0.13.0 or 1.0.0?

**Comment by danmilon** [comment]
thanks! I think we need a release as well?

**Pull request merged**